doc/source/docs/user_guide/io.rst

```diff
@@ -177,12 +177,12 @@ Skip loading geometry from the file:
     )
 
 
-SQL WHERE Filter
-^^^^^^^^^^^^^^^^^
+SQL WHERE filter
+^^^^^^^^^^^^^^^^
 
 .. versionadded:: 0.12
 
-Load in a subset of data with a `SQL WHERE clause <https://gdal.org/user/ogr_sql_dialect.html#where>`__.
+Load in a subset of data with a SQL `WHERE <https://gdal.org/user/ogr_sql_dialect.html#where>`_ clause.
```

Comment:
Very much a nitpick, but following the last bullet point at https://developers.google.com/style/link-text#write-link-text (about `--hostname flag`), one could argue it recommends to include "clause" in the link text.